cluster/manifests/skipper/deployment-a.yaml

```diff
@@ -66,10 +65,10 @@ spec:
           - "-opentracing=lightstep component-name=skipper-ingress token=$(LIGHTSTEP_TOKEN) collector=tracing.zmon.zalan.do:8444 cmd-line=skipper-ingress max-buffered-spans=4096"
         resources:
           limits:
-            memory: {{ .ConfigItems.skipper_limits_mem }}
+            memory: {{ .ConfigItems.skipper_ingress_cpu }}
```

Comment:
This should be `ConfigItems.skipper_ingress_memory` instead of `cpu`. (Same for `b` and `c`).